### DISCUSSION TITLE
Catalogue failing to load items on frontend

### DISCUSSION THREAD

**Discussion by dam1212** [learndevopsonline/learndevopsonline#1214] [Q&A]
[Doc1.docx](https://github.com/learndevopsonline/learndevopsonline/files/12113922/Doc1.docx)

**Comment by maheshbusireddy** [comment]
Hi before you are going to post a discussion in git hub. Once go through this video https://www.youtube.com/watch?v=scWcQyddWfA 

Coming to your question Servers are not reachable to  check the issue. When servers are up and running then only we are able to track your issue. So you should maintain your servers in running state.
